frontend/src/metabase/query_builder/components/view/ExploreResultsLink.tsx

```diff
@@ -0,0 +1,25 @@
+import { t } from "ttag";
+import Link from "metabase/core/components/Link";
+import ViewButton from "metabase/query_builder/components/view/ViewButton";
+import type Question from "metabase-lib/Question";
+import { getUrl as ML_getUrl } from "metabase-lib/urls";
+
+type Props = {
```

Comment:
`interface ExploreResultsLinkProps`

Reply:
If it's not exported, why would we need that?